src/cli/README_SRP.md

```diff
@@ -0,0 +1,181 @@
+# OpenThread CLI - SRP (Service Registration Protocol)
+
+## Quick Start
+
+### Build with SRP Server & Client support
+
+Use the `SRP_SERVER=1 SERVICE=1 ECDSA=1` build switches to enable SRP Server API support, and the `SRP_CLIENT=1 ECDSA=1` build switches for SRP Client API support.
+
+Make simulation SRP Server & Client:
+
+```bash
+./bootstrap
+make -f examples/Makefile-simulation SRP_SERVER=1 SERVICE=1 SRP_CLIENT=1 ECDSA=1
+```
+
+## Simulation Example
+
+### Start SRP Server
+
+Start the SRP Server node:
+
+```bash
+./output/simulation/bin/ot-cli-ftd 1
+```
+
+Setup a Thread network and start the SRP Server:
+
+```bash
+> panid 0xface
+Done
+> channel 18
+Done
+> masterkey 00112233445566778899aabbccddeeff
+Done
+> ifconfig up
+Done
+> thread start
+Done
+> state
+leader
+Done
+>ipaddr
+fd00:db8:0:0:0:ff:fe00:fc10
+fd00:db8:0:0:0:ff:fe00:fc00
+fd00:db8:0:0:0:ff:fe00:8400
+fd00:db8:0:0:7cdf:b15e:30e0:df4e
+fe80:0:0:0:146e:a00:0:1
+Done
+> srp server enable
+Done
+> netdata show
+Prefixes:
+Routes:
+Services:
+44970 5d c002 s 8400
+Done
+```
+
+The SRP Server listening UDP port which is `c002`(`49154`) is included in the Server Data (listed by the `netdata show` command).
+
+### Start SRP Client
+
+Start the SRP Client node:
+
+```bash
+./output/simulation/bin/ot-cli-ftd 2
+```
+
+Join the Thread Network and register a `_ipps._tcp` service:
+
+```bash
+> panid 0xface
```

Comment:
Prefer to only specify master key - see [src/cli/README_DATASET.md#attach-to-existing-network](https://github.com/openthread/openthread/blob/master/src/cli/README_DATASET.md#attach-to-existing-network).